### PR TITLE
Fix typo in filter.py

### PR DESCRIPTION
**Pull request by MPvHarmelen**

### CODE REVIEW

**Comment by agramfort** [comment]
sorry I don't see the typo you fixed

File: mne/filter.py [hunk 1]
```diff
@@ -1241,10 +1241,10 @@ def _mt_spectrum_proc(x, sfreq, line_freqs, notch_widths, mt_bandwidth,
     for rm_freqs in freq_list:
         if line_freqs is None:
             if len(rm_freqs) > 0:
-                logger.info('Detected notch frequencies:\n%s'
-                            % ', '.join([str(rm_f) for rm_f in rm_freqs]))
+                found_freqs = ', '.join([str(rm_f) for rm_f in rm_freqs]))
             else:
-                logger.info('Detected notch frequecies:\nNone')
```
Comment (larsoner) [inline mne/filter.py:1247]: frequecies was the fixed one

File: mne/filter.py [hunk 2]
```diff
@@ -1241,10 +1241,10 @@ def _mt_spectrum_proc(x, sfreq, line_freqs, notch_widths, mt_bandwidth,
     for rm_freqs in freq_list:
         if line_freqs is None:
             if len(rm_freqs) > 0:
-                logger.info('Detected notch frequencies:\n%s'
-                            % ', '.join([str(rm_f) for rm_f in rm_freqs]))
+                found_freqs = ', '.join([str(rm_f) for rm_f in rm_freqs]))
```
Comment (larsoner) [inline mne/filter.py:1244]: ... but this introduced a SyntaxError because of the `))` instead of `)` at the end :)

**Comment by larsoner** [comment]
Thanks @MPvHarmelen !

**Pull request merged**